### PR TITLE
Message type change.

### PR DESCRIPTION
**Pull request by Khan1988**
People tracker now listens to PeopleStamped message rather than people message.
PeopleStamped message is an array of PersonStamped message type.

### CODE REVIEW

**Comment by gpdas** [comment]
@Khan1988 in `bayes_people_tracker_logging` (?)
```13:50:06 CMake Error at /opt/ros/kinetic/share/genmsg/cmake/genmsg-extras.cmake:263 (message):
13:50:06   Messages depends on unknown pkg: people_msgs (Missing
13:50:06   'find_package(people_msgs)'?)
13:50:06 Call Stack (most recent call first):
13:50:06   CMakeLists.txt:29 (generate_messages)```

File: bayes_people_tracker/src/bayes_people_tracker/people_tracker.cpp [hunk 1]
```diff
@@ -627,7 +627,7 @@ void PeopleTracker::connectCallback(ros::NodeHandle &n) {
         for(it = subscribers.begin(); it != subscribers.end(); ++it)
             subscribers[it->first] = n.subscribe<geometry_msgs::PoseArray>(it->first.second.c_str(), 1000, boost::bind(&PeopleTracker::detectorCallback, this, _1, it->first.first));
         for(it = subscribers_people.begin(); it != subscribers_people.end(); ++it)
-            subscribers_people[it->first] = n.subscribe<people_msgs::People>(it->first.second.c_str(), 1000, boost::bind(&PeopleTracker::detectorCallback_people, this, _1, it->first.first));
+            subscribers_people[it->first] = n.subscribe<bayes_people_tracker::PeopleStamped>(it->first.second.c_str(), 1000, boost::bind(&PeopleTracker::detectorCallback_people, this, _1, it->first.first));
```
Comment (marc-hanheide) [inline bayes_people_tracker/src/bayes_people_tracker/people_tracker.cpp]: Sorry, but this change breaks the system for many other projects, as it changes the type of a subscriber. That means that all our other projects will not work with this anymore, as they are publishing `people_msgs::People`. I must insist you create a separate subscriber for this, not change an existing one in an non-compatible way.

**Review by marc-hanheide — CHANGES_REQUESTED** [review]
breaks interface, cannot be accepted as is.

File: bayes_people_tracker/src/bayes_people_tracker/people_tracker.cpp (hunk 1 above)
Comment (Khan1988) [reply bayes_people_tracker/src/bayes_people_tracker/people_tracker.cpp]: Fixed now with separate subscriber for peoplestamped msg type.

**Pull request merged**